### ISSUE TITLE
Fatal error after after deleting a page of products 

### ISSUE THREAD

**Issue by Sillky01** [opencart/opencart#5403]
Hi  can any one help help me ? 
 I am new to open cart and have caused an error by  deleting a page of products using the top select all tick box. It was the first page in my catalogue. 
I now have a fatal error Allowed memory size of 67108864 bytes exhausted (tried to allocate 26008 bytes) in /home/dir1/public_html/subdir1/system/library/image.php on line 34
I ran a repair over the categories and have also run a repair of the sql database. I also increased the memory size  in both php.ini files

The error occurs when I navigate to the site page and also in opencart admin when I drill down to product from the menu 

I am using open cart 1.5.6. and preparing to upgrade to the latest version

**Comment by Sillky01** [comment]
Since posting I have restored the database and can delete individual items and some pages however not the first  page that caused the original error( it still causes the error above as described) . 
Unfortunately I get the same problem on other pages which I dont have backups for so restoring isn't enough to get the site working. I have tried adding an item to the category not loading but same error. Any advice would be really appreciated.

**Comment by danielkerr** [comment]
you have an image to large for php to resize 

login via ftp and delete the largest image

**Comment by Sillky01** [comment]
Thank you so much for helping me :0) .his error only occured after I dleted a page of products. it worked before. 
I have deleted any large files - adjusted  .htaccess to 256m and the php.ini's to 256m as in faq's suggestion.
The error from logs today is 2017-05-02 0:19:14 - PHP Notice:  Undefined index: min_price in /home/jul1e/public_html/domestic-furniture/catalog/controller/module/filterpro.php on line 440
2017-05-02 0:19:14 - PHP Notice:  Undefined index: max_price in /home/jul1e/public_html/domestic-furniture/catalog/controller/module/filterpro.php on line 441
